product_code_builder/models/product_attribute_value.py

```diff
@@ -0,0 +1,18 @@
+# coding: utf-8
+# © 2015 Benoît GUILLOT <benoit.guillot@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ProductAttributeValue(models.Model):
+    _inherit = "product.attribute.value"
+    _code = "code"
```

Comment:
This is a mistake https://github.com/OCA/product-attribute/commit/726c16ec05b0c65d542322fe529e551edf057ab5
it must be _order \= 'code' . Sorry for that